lightbulb/command_handler.py

```diff
@@ -169,6 +175,27 @@ def __init__(
 
         self._help_impl = help_class(self)
 
+        self.extensions_path = extensions_path
+        if self.extensions_path:
+            self.full_extensions_path = pathlib.Path.cwd() / self.extensions_path.replace(".", os.sep)
```

Comment:
Using `.` is a bit counter intuitive if you ask for a system path

Reply:
Consistent with the argument `lightbulb.Bot.load_extension` takes

Reply:
You are providing a folder for watchgod to check in, not the extension to load. I think it might be best to do a system path, not python import path. Up to @tandemdude